blogs/scripts/scripts.js

```diff
@@ -571,6 +571,133 @@ export async function addBackOfficeMetaTags() {
   }
 }
 
+/**
+ * loads a block named 'header' into header
+ */
+
+async function loadKeysightHeader(header) {
+  const jqueryScript = document.createElement('script');
+  jqueryScript.type = 'text/javascript';
+  jqueryScript.innerHTML = '';
+  jqueryScript.src = 'https://ajax.googleapis.com/ajax/libs/jquery/3.5.0/jquery.min.js';
+  document.head.appendChild(jqueryScript);
+  function getCookie(name) {
```

Comment:
in the video block js, there is already a function for reading a cookie value. May want to move that function up to scripts.js and use that instead